app/views/admin/editions/show/_sidebar_link_checker.html.erb

```diff
@@ -0,0 +1,73 @@
+<% if report.new_record? %>
+  <div class="app-view-edition-summary__sidebar-info">
+    <p class="govuk-body">Check this document for broken links. The report will take a few moments to complete.</p>
+    <%= render partial: 'admin/editions/show/sidebar_link_checker_form', locals: {
+      reportable: report.link_reportable,
+      button_text: 'Check for broken links',
+      margin_bottom: 6
+    } %>
+  </div>
+<% elsif report.in_progress? %>
+  <div class="app-view-edition-summary__sidebar-info">
+    <%= image_tag('admin/ajax_loader_blue_32.gif', width: 24, height: 24, class: 'pull-left add-right-margin') %>
+    <%= render 'admin/editions/show/sidebar_link_checker', reportable: report.link_reportable, button_text: 'Check for broken links' %>
+    <p class="govuk-body">Broken link report in progress.<br />Please refresh the page to check progress.</p>
+  </div>
+<% elsif report.broken_links.any? || report.caution_links.any? %>
+  <%= render "components/inset_prompt", {
+    title: t("broken_links.title"),
+    description: capture do %>
+      <% report.links.sort_by(&:status).group_by(&:status).each do |status, links| %>
+        <% next unless %w(broken caution).include? status %>
+
+        <p class="govuk-body"><%= t "broken_links.#{status}.subheading" %></p>
+
+        <ul class="govuk-list">
+          <% links.each do |link| %>
+            <li>
+              <%= link_to link.uri.truncate(50), link.uri, title: link.uri, class: 'govuk-link' %>
```

Comment:
Are we happy with the link moving out of the inset component? 

<img width="446" alt="image" src="https://user-images.githubusercontent.com/42515961/205270170-be665005-bd85-4cf3-8d08-bb785fc1c162.png">

Reply:
I think that this has already been fixed
https://github.com/alphagov/whitehall/pull/7083/commits/3da0aac35bd0776b5df18277ce4cb484398751fb#diff-1c67913b41e03a4fb87298119742d00de1f3fa686f195d9a793be6c41de7a869R57

Reply:
Oh okay i think the class needs to be added the broken links list then as its not being applied at the moment

Reply:
Ah, I dropped the class from the DOM at some point
Have added it back